repose-aggregator/artifacts/docker/build.gradle

```diff
@@ -0,0 +1,53 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+import com.bmuschko.gradle.docker.tasks.image.DockerRemoveImage
+import com.bmuschko.gradle.docker.tasks.image.DockerPushImage
+
+apply plugin: 'com.bmuschko.docker-remote-api'
+
+// This forces the usage of a more recent unix-socket-factory which makes Mac OS X happy
+// It also forces the usage of jackson dependencies which were not being brought in as expected
+// TODO: Remove unix-socket-factory after upgrading to gradle-docker-plugin 3.0.4
+configurations {
+    dockerJava {
+        resolutionStrategy {
+            force 'de.gesellix:unix-socket-factory:2016-04-06T22-21-19'
+            force 'com.fasterxml.jackson.core:jackson-databind:2.6.4'
+            force 'com.fasterxml.jackson.core:jackson-core:2.6.4'
+            force 'com.fasterxml.jackson.core:jackson-annotations:2.6.4'
+        }
+    }
+}
+
+String reposeVersion = project.hasProperty('repose-version') ? project.property('repose-version') as String : version as String
```

Comment:
It enabled doing what I did a couple of days ago -- releasing Docker Hub images for versions of Repose that we've already cut. Or for versions that we patch (that came before these tasks).